recipe/meta.yaml

```diff
@@ -19,16 +19,22 @@ outputs:
       run_exports:
         strong:
           - cudatoolkit {{ cuda_compiler_version }}|{{ cuda_compiler_version }}.*
+          - __glibc >=2.17  # [cuda_compiler_version == "11.0"]
     requirements:
       host:
         # Needed to symlink libcuda into sysroot libs.
         - {{ compiler("c") }}
+        - sysroot_linux-64 2.17  # [cuda_compiler_version == "11.0"]
+        - __glibc >=2.17         # [cuda_compiler_version == "11.0"]
       run:
+        - sysroot_linux-64 2.17  # [cuda_compiler_version == "11.0"]
+        - __glibc >=2.17         # [cuda_compiler_version == "11.0"]
```

Comment:
Does this address your concerns, @beckermr?

Reply:
It does since this is required to actually use the package in that case. However, @isuruf didn't want this constraint since he wants to use clang w/ some of the stuff in this package but not all of it. For that case, I would think we'd want a separate cuda-only activation script that the nvcc one would depend on.

Reply:
Again I don't really care. I am trying to make sure I understand what is going on.

Reply:
Having a separate package could make sense.

That said, CentOS 6 reaches its [EOL 30 Nov 2020]( https://wiki.centos.org/About/Product ). So in a couple months. I'm not sure how much longer we plan to support it, but maybe this becomes a non-issue in the near future.

Reply:
+1 to having a separate cuda activation script package. From the outside, the point of this package is to get the `nvcc` compiler. This package handling the activation script things is an implementation detail that could change at any point in the future as long as `nvcc` is still being delivered in some way.